### DISCUSSION TITLE
Status of girder-worker

### DISCUSSION THREAD

**Discussion by Xarthisius** [girder/girder#3589] [Support] [answered]
I noticed that https://github.com/girder/girder_worker was archived on 11/07/24. Is there a replacement / alternative to celery tasks playing nicely with girder-jobs?

**Comment by zachmullen** [comment, marked as answer]
The package itself will still be available as normal, but we've moved it inside of the girder/girder repository, which is happening on the `v4-integration` branch.

**Reply by Xarthisius** [reply]
AFAICT only the girder server-side part of girder_worker was moved to plugins/worker (basically what was `girder_plugin` in the original repo). What about the part that was modifying celery components? Like attaching girder client and job_manager to the task, updating job status to match task status etc?

OH, it's in girder/girder:worker. I'm blind. sorry!
